.github/workflows/sanity.yml

```diff
@@ -19,3 +19,13 @@ jobs:
 
     - name: run misspell
       run: ./bin/misspell -error .
+
+  encoding:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: check out code
+      uses: actions/checkout@v2
+
+    - name: detect non-ASCII encoding and trailing space
+      run: python3 ./build/sanity.py
```

Comment:
Can you think of alternate name other than just `sanity` ? `sanitycheck` sounds more description to me than just `sanity` .

Reply:
I've got no strong opinion about this. Changing now.